app/views/posts/index.html.erb

```diff
@@ -1,2 +1,22 @@
-<h2>Post#index</h2>
-<p>Here are ALL Posts for a given user you are in '/users/user_id/posts'</p>
+<% content_for :title, "#{@user.name}'s Posts " %>
+<%= render "users/user", user: @user %>
+<% @user.posts.each do | post | %>
+  <%= link_to user_post_path(@user, post) do %>
+    <%= render "post", post: post %>
+    <div class="user-post-comments mt-2">
+      <h3 class="text-xl">
+        <span class="underline decoration-wavy decoration-slate-500">Comments</span>
+        <span>For Post: </span>
+        <span class="text-blue-gray-600 underline decoration-red-400"><%= "#{post.title}"%></span>
+      </h3>
+      <% post.recent_five.each do |comment| %>
+        <div class='post-comment'>
+          <p>
+            <span><strong><%= comment.author.name %></strong></span>
+            <span><%= comment.text %></span>
+          </p>
+        </div>
+      <%end%>
+    </div>
+  <%end%>
+<%end%>
```

Comment:
- [ ] Kindly improve the post index layout to match the wireframe. You need to improve the user information section and add margins on the left and right of the page to match [the wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACBAEJCZ2O6374JSUN6DQSU4ZBNG6ZA)

  ![image](https://user-images.githubusercontent.com/36542986/230017887-dd6f5e08-a434-4af9-a360-c8bd4f96faed.png)